rest-api/src/main/resources/definitions/weekly_adherence_report_row.yml

```diff
@@ -0,0 +1,42 @@
+type: object
+description: Properties of the entries in the array under each day of the weekly adherence report (0-6). These row descriptions are in the same position as the entries they describe in these arrays (when there is no entry for that position in the array, there is an empty EventStreamDay entry with an empty timeWindows array).
+properties:
+    label:
+        type: string
+        readOnly: true
+    searchableLabel:
+        type: string
+        readOnly: true
+    sessionGuid:
+        type: string
+        readOnly: true
+        x-nullable: false
+    sessionInstanceGuid:
+        type: string
+        readOnly: true
+        x-nullable: false
```

Comment:
I don't see sessionInstanceGuid in the report row in BridgeServer2. Should this be here in the SDK?

Reply:
removed